pythonAPIClient/client.py

```diff
@@ -0,0 +1,73 @@
+from .camera import Camera, IPCamera, NonIPCamera, StreamCamera
+from .error import Error
+import requests
+
+class Client(object):
+    """
+    Represent a CAM2 client application.
+    """
+    # TODO: corresponding to the auth route
+    @staticmethod
+    def request_token(id, secret):
+        return 'dummy'
+
+    # TODO: set authentication in header
+    @staticmethod
+    def header_builder(token):
+        return 'dummy'
+
+    def __init__(self, id, secret, token=None):
+        self.id = id
+        self.secret = secret
+        if token is None: 
```

Comment:
I don't think there's particularly a problem doing this here, but we should just think about if this is necessary or not for the constructor.

Reply:
do you mean token?